src/main/java/org/broadinstitute/dsde/consent/ontology/datause/api/LuceneOntologyTermSearchAPI.java

```diff
@@ -79,7 +78,13 @@ private void initAPI() throws OWLOntologyCreationException, IOException {
                     OWLAnnotationProperty def = annotationProperties.get(FIELD_DEFINITION_CLASS);
                     OWLAnnotationProperty deprecated = annotationProperties.get("deprecated");
 
-                    ontology.classesInSignature().forEach(owlClass -> {
+                    Set<OWLClass> owlClasses = ontology.classesInSignature().collect(Collectors.toSet());
+                    owlClasses.addAll(ontology.
+                            directImports().
+                            flatMap(HasClassesInSignature::classesInSignature).
+                            collect(Collectors.toSet()));
```

Comment:
This is the solution, pulling all direct imports into the cache.

Everything else is mostly just formatting changes.